server/db/migrations/patch-2-3.sql

```diff
@@ -1,2 +0,0 @@
--- used for batch reindexing based on creation time
-CREATE INDEX user_pages_created_at ON user_pages (created_at);
```

Comment:
Wait, can we delete migrations? Or don't we care yet since we aren't in prod and this is all just local and we're probably deleting the database each time anyways?